Cargo.toml

```diff
@@ -33,6 +33,7 @@ libm    = [ "simba/libm" ]
 libm-force = [ "simba/libm_force" ]
 macros = [ "nalgebra-macros" ]
 cuda   = [ "cust_core", "simba/cuda" ]
+rayon-par  = [ "std", "rayon" ]
```

Comment:
Completely agree this would be more elegant. I tried this last time and I can achieve that by writing the dependency as `"dep:rayon"`. Last PR this broke the test on one of the architectures. Most likely because the compiler was out of date. If it's ok to bump the MSRV for that feature I'd also prefer the more elegant way of doing that. Let me know what you think

Reply:
Using `dep:rayon` isn’t possible right now since the `cuda` version requires a very specific (older) version of the compiler that doesn’t understand that annotation. Nonetheless, we don’t need to make the `par-iter` imply `std` (because `nalgebra` itself won’t need the `std` feature enabled to compile with `par-iter` enabled, and `rayon` itself will have `std` enabled which it needs to wore).

So I suggest we just use `rayon` as the feature name (and don’t declare explicitly a feature like `par-iter`).